MobileBuy/sample/src/shopify/java/com/shopify/sample/data/graphql/Query.java

```diff
@@ -0,0 +1,53 @@
+package com.shopify.sample.data.graphql;
+
+import com.shopify.buy3.Storefront;
+
+public class Query {
```

Comment:
Is it used somewhere?

Reply:
It is used in `com.shopify.sample.domain.usecases.FetchCollectionsUseCaseImpl`